specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json

```diff
@@ -0,0 +1,848 @@
+          },
+          {
+            "$ref": "#/parameters/QueryApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/Operation"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Operation": {
+            "$ref": "./examples/get_operation.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "BatchSynthesis": {
+      "description": "Bath synthesis task.",
+      "required": [
+        "inputKind",
+        "avatarConfig"
+      ],
+      "type": "object",
+      "properties": {
+        "inputKind": {
+          "$ref": "#/definitions/InputKind"
+        },
+        "inputs": {
+          "description": "Text or SSML inputs, it's allowed up to 1000 items.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BatchSynthesisInput"
+          }
+        },
+        "synthesisConfig": {
+          "$ref": "#/definitions/BatchSynthesisConfig"
+        },
+        "customVoices": {
+          "description": "The custom voice map of voice name and deployment ID .",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9][a-zA-Z0-9._-]{1,62}[a-zA-Z0-9]$",
+            "minLength": 3,
+            "maxLength": 64
+          }
+        },
+        "properties": {
+          "$ref": "#/definitions/BatchSynthesisProperties"
+        },
+        "avatarConfig": {
+          "$ref": "#/definitions/BatchAvatarConfig"
+        },
+        "outputs": {
+          "$ref": "#/definitions/BatchSynthesisOutputs",
+          "readOnly": true
+        },
+        "status": {
+          "$ref": "#/definitions/Status",
+          "readOnly": true
+        },
+        "lastActionDateTime": {
+          "description": "The time-stamp when the current status was entered.",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "$ref": "#/definitions/ResourceId",
+          "readOnly": true
+        },
+        "description": {
+          "description": "The description of the batch synthesis task.",
+          "type": "string"
+        },
+        "createdDateTime": {
+          "description": "The time-stamp when the object was created.",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "BatchSynthesisBillingDetails": {
+      "description": "Bath synthesis billing details.",
```

Comment:
typo Bath

Please double check for other places.

Reply:
oops. :) thanks for reminding, fixed